docs/Hooks.md

```diff
@@ -278,12 +280,12 @@ hooks:
     type: slackdiff
     events: [post_store]
     token: SLACK_BOT_TOKEN
-    channel: "#network-changes"
+    channel: "CHANNEL_ID"
     diff: false
     message: "%{node} %{group} %{model} updated https://git.intranet/network-changes/commit/%{commitref}"
 ```
 
-Note the channel name must be in quotes.
+Note the channel ID must be in quotes.
```

Comment:
The channel ID does not need to be in quotes as they don't use the # symbol (or any other YAML reserved symbol)